.github/workflows/pr504-canvas-parity.yml

```
@@ -0,0 +1,127 @@
+              'chart resolver import'
+          )
+
+          replace_once(
+              '    private val chartsRepository = ChartsRepository(application.applicationContext)\n    private val officialArtworkRepository = OfficialArtworkRepository(application.applicationContext)\n',
+              '    private val chartsRepository = ChartsRepository(application.applicationContext)\n    private val motionTrackMetadataResolver = ChartOfficialArtworkResolver(application.applicationContext)\n    private val officialArtworkRepository = OfficialArtworkRepository(application.applicationContext)\n',
+              'motion metadata resolver field'
+          )
+
+          marker = '    private fun refreshMotionArtworkAround(current: Track) {\n'
+          helper = '''    private suspend fun motionArtworkLookupTrack(track: Track): Track {
+        val snapshot = _state.value
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)